pkg/clients/aws/client.go

```diff
@@ -84,18 +92,32 @@ func (c *client) getAccountNumber(ctx context.Context) (string, error) {
 }
 
 var (
-	productSKUField         = "ProductSKU"
-	rancherProductSKU       = "0b87d4fa-d1fe-41d8-830b-67d4ec381549"
-	maxResults        int32 = 1
+	productSKUField                = "ProductSKU"
+	rancherProductSKUAmea          = "0b87d4fa-d1fe-41d8-830b-67d4ec381549"
+	rancherProductSKUNonAmea       = "d3ca94f5-fb96-44c1-a094-415fc0463605"
+	maxResults               int32 = 1
 )
 
 func (c *client) GetRancherLicense(ctx context.Context) (*types.GrantedLicense, error) {
+	license, err := c.getLicenseForProductID(ctx, rancherProductSKUAmea)
+	if err != nil {
+		// if we could not get the original license, attempt to retrieve the license for nonAMEA countries
+		license, newErr := c.getLicenseForProductID(ctx, rancherProductSKUNonAmea)
+		if newErr != nil {
+			return nil, fmt.Errorf("%w, unable to get license for non-amea: %s", err, newErr.Error())
```

Comment:
This could be reworded so that the users can easily understand that amea also failed.
```suggestion
			return nil, fmt.Errorf("unable to get license for amea: %w, and non-amea: %s", err, newErr.Error())
```

Reply:
Good suggestion, thanks. I'll also remove the first wrap. I think it doesn't make sense to wrap one error and not the other, especially when the code is ambiguous as to which error "matters"